consensus/src/quorum_store/quorum_store_db.rs

```diff
@@ -59,6 +63,15 @@ impl QuorumStoreDB {
 
         Self { db }
     }
+
+    /// Relaxed writes instead of sync writes.
+    pub fn put<S: Schema>(&self, key: &S::Key, value: &S::Value) -> Result<(), DbError> {
+        // Not necessary to use a batch, but we'd like a central place to bump counters.
+        let mut batch = self.db.new_native_batch();
```

Comment:
what's the difference between native batch and schema batch?

Reply:
I just copied the code and changed the write function lol